owncloudApp/src/main/res/drawable/ic_baseline_lock_reset_24.xml

```diff
@@ -0,0 +1,5 @@
+<vector android:height="24dp" android:tint="#000000"
+    android:viewportHeight="24" android:viewportWidth="24"
+    android:width="24dp" xmlns:android="http://schemas.android.com/apk/res/android">
+    <path android:fillColor="@android:color/white" android:pathData="M13,3c-4.97,0 -9,4.03 -9,9H1l4,4l4,-4H6c0,-3.86 3.14,-7 7,-7s7,3.14 7,7s-3.14,7 -7,7c-1.9,0 -3.62,-0.76 -4.88,-1.99L6.7,18.42C8.32,20.01 10.55,21 13,21c4.97,0 9,-4.03 9,-9S17.97,3 13,3zM15,11v-1c0,-1.1 -0.9,-2 -2,-2s-2,0.9 -2,2v1c-0.55,0 -1,0.45 -1,1v3c0,0.55 0.45,1 1,1h4c0.55,0 1,-0.45 1,-1v-3C16,11.45 15.55,11 15,11zM14,11h-2v-1c0,-0.55 0.45,-1 1,-1s1,0.45 1,1V11z"/>
+</vector>
```

Comment:
Are both `lock_reset` icons actually needed? Or do we only use one?

Reply:
Actually we don't use any of them, it is a change of icon i would like to propose but you are right, we will only need one of those

Reply:
Ok, nice! But if you want to propose it, do it, so that we don't have this icon with no uses right now 😉

Reply:
Finally I decided to delete it cause i had no time for more even though i woould have loved proposing it 😔